app/theme.js

```diff
@@ -25,4 +26,21 @@ export default {
     heading: `Inter,${theme.fonts.heading}`,
     body: `Inter,${theme.fonts.body}`,
   },
+  icons: {
+    ...theme.icons,
+    customCheck: {
+      path: (
+        <svg viewBox="0 0 512 512">
+          <path
+            fill="#31A7D8"
+            d="M512 58.668C512 26.305 485.695 0 453.332 0H58.668C26.305 0 0 26.305 0 58.668v394.664C0 485.695 26.305 512 58.668 512h394.664C485.695 512 512 485.695 512 453.332zm0 0"
+          />
+          <path
+            fill="#fafafa"
+            d="M385.75 171.586c8.34 8.34 8.34 21.82 0 30.164L247.082 340.414c-4.16 4.16-9.621 6.254-15.082 6.254s-10.922-2.094-15.082-6.254l-69.332-69.332c-8.344-8.34-8.344-21.824 0-30.164 8.34-8.344 21.82-8.344 30.164 0l54.25 54.25 123.586-123.582c8.34-8.344 21.82-8.344 30.164 0zm0 0"
+          />
+        </svg>
```

Comment:
Icon should be pasted only after minimizing with [SVGO](https://jakearchibald.github.io/svgomg/)
Also we only need the check icon not the whole box
```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 7"><path fill="none" fill-rule="evenodd" stroke="#FFF" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M1.5 3.788L3.651 6l5.058-5"/></svg>
```

as below

```json
icons: {
    ...theme.icons,
    customCheck: {
      path: (
        <path
          fill="none"
          fillRule="evenodd"
          stroke="#FFF"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1.5"
          d="M1.5 3.788L3.651 6l5.058-5"
        />
      ),
      viewBox: '0 0 10 7',
    },
  },
```

Reply:
We only need the check svg not the box as well.
See my path, how simple it and how I have declared the viewbox.
That will give the checkbox correct style.

Reply:
![image](https://user-images.githubusercontent.com/39694575/77744018-f772ca80-703e-11ea-835b-129e6447e49b.png)